pnpm-lock.yaml

```diff
@@ -255,7 +255,7 @@ importers:
       '@reactioncommerce/logger': link:../../packages/logger
       '@reactioncommerce/nodemailer': 5.0.5
       '@reactioncommerce/random': link:../../packages/random
-      '@snyk/protect': 1.1061.0
```

Comment:
Generally I think you should avoid pulling in these Snyk updates so that you can avoid unnecessary merge conflicts uypstream